
time Stamp
bool HardwareOK
string Firmware
string Hardware
float32 Voltage
float32 Current
float32 InternalTemp
float32 ExternalTemp
bool Fault_PWM_1
bool Fault_PWM_2
bool Fault_PWM_3
uint8[12] PwmDuty
